docs/design/chart_format.md

```diff
@@ -189,25 +176,14 @@ keywords:
 * proxy
 source: https://github.com/foo/bar
 home: http://nginx.com
-depends:
-        charts:
-          - name: memcached
-                location: helm.sh/charts/memcached
-                version: 0.9.3
-        kubernetes:
-                version: “>=1.0.0”
-                extensions:
-                  - extensions/v1beta1
+expander:
+  name: GoTemplates
```

Comment:
The expander is mandatory in the current implementation, so no.  I suppose we could modify the behavior such that if no expander is given in the chart, the chart is simply not expanded.  However this still takes some code to parse it as YAML, handle the errors, and no code exists in manager to do that right now.